cli/src/pcluster/models/imagebuilder.py

```diff
@@ -368,6 +422,53 @@ def _upload_artifacts(self):
                 f"Unable to upload imagebuilder cfn template to the S3 bucket {self.bucket.name} due to exception: {e}"
             )
 
+    def delete(self, force=False):
+        """Delete CFN Stack and associate resources and deregister the image."""
+        # validate image name
+        self._validate_image_name()
```

Comment:
We shouldn't validate image name at deletion time. If the image is created with a long(wrong?) name we need a way to delete it.
I think image name validation at creation time is enough.

Reply:
done